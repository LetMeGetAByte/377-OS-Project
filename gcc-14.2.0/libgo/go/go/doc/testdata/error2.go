version https://git-lfs.github.com/spec/v1
oid sha256:75c66e0e7123ad7219e4385710ce9852c37a0c1603314fcb703db08e8fe8556f
size 618
