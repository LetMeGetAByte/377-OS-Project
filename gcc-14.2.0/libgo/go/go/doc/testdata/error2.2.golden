version https://git-lfs.github.com/spec/v1
oid sha256:167f3f4687187c4bc468ab20e6f8442a5d61fe530a82453b678ae12d08a28564
size 337
